app/components/post_list/post/index.ts

```diff
@@ -149,7 +149,7 @@ const withPost = withObservables(
             isConsecutivePost,
             isEphemeral,
             isFirstReply: of$(isFirstReply(post, previousPost)),
-            isFlagged,
+            isFlagged: isSaved,
```

Comment:
shouldn't this be just?

```suggestion
            isSaved,
```